.github/workflows/ci.yml

```diff
@@ -13,29 +13,47 @@ jobs:
     steps:
       - name: Clone This Repo
         uses: actions/checkout@v2
-      - name: Build
+      - name: Build Project
         run: |
-          sudo apt-get install -y lcov
-          cmake -S test -B build/ \
+          CFLAGS="  --coverage -O0 -Wall -Wextra -Werror"
+          CFLAGS+=" -DFORTIFY_SOURCE=1"
+          CFLAGS+=" -fsanitize=address"
+          CFLAGS+=" -fsanitize=pointer-compare -fsanitize=pointer-subtract"
+          CFLAGS+=" -fsanitize=undefined"
+          CFLAGS+=" -fsanitize-recover=undefined,float-cast-overflow,float-divide-by-zero,bounds-strict"
+          cmake -S test -B build \
           -G "Unix Makefiles" \
           -DCMAKE_BUILD_TYPE=Debug \
-          -DCMAKE_C_FLAGS='--coverage -Wall -Wextra -DNDEBUG'
-          make -C build/ all
-      - name: Test
+          -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON \
+          -DCMAKE_C_FLAGS="${CFLAGS}"
+          make -C build all
+      - name: Run with Address Sanitizer
         run: |
-          cd build/
+          pushd build
```

Comment:
Is there a reason for changing this? Seems to do same thing. It would be better to leave it the same as in all the other libraries, so the differences are just whats actually different.j

Reply:
It's a bit cleaner.. thats all.

Reply:
I disagree, and I'd rather not introduce unnecessary bashisms, especially when the posix way is equally simple. That and I would rather we keep the same for maintainability of all the ci.ymls

Reply:
ok.